graphics/ctpv/Portfile

```
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github   1.0
+PortGroup           makefile 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+github.setup        NikitaIvanovV ctpv 41507944c49c4bbbf8f3a21dde42243c6b4e1b79
+github.tarball_from archive
+
+categories          graphics
+license             MIT
+platforms           darwin linux
+maintainers         {icloud.com:github.ssk @suhailskhan} \
+                    openmaintainer
+description         Image previews for lf file manager
+long_description    ctpv is a previewer utility for terminals. It supports previews for \
+                    source code, archives, PDF files, images and videos.
+
+checksums           rmd160  d20b6aa22601d1e946e05903a23c6738c82da36f \
+                    sha256  118c473d20aa98089eed7cf94d4476f647ca90d65b7449f7f60e7159a833c321 \
+                    size    2010651
+
+depends_lib-append  lib:libcrypto:openssl \
```

Existing ports do not use `lib:libcrypto:…` and so maybe ctpv should not either. There are a handful of ports which use `path:lib/libcrypto.dylib:…` instead, presumably to avoid non-MacPorts libcrypto. Maybe there is a better alternative which I’m not aware of, e.g. something in the openssl portgroup.
```suggestion
depends_lib-append  path:lib/libcrypto.dylib:openssl \
```

---

I see. The dependencies reported by MacPorts on my system are unaffected by the change, so it seems it’s safe.